karma.conf.js

```diff
@@ -21,6 +21,12 @@ module.exports = function (config) {
       fixWebpackSourcePaths: true
```

Comment:
if you don't have any UI tests all this stuff just adds to your build & deploy time so consider removing it